virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,29 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 //https://bugzilla.redhat.com/show_bug.cig?=id=1959237
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* When creating virtual machines using common-templates or the virtual machine wizard, the default `terminationGracePeriodSeconds` value is set to 0 (force stop).
```

Comment:
s/creating/you create

s/using/by using

Question: should 0 be in backticks if you're referring to the value in the yaml?

Suggestion: We generally don't use parenthesis. Instead of `(force stop)`, could you very briefly explain why 0 = force stop?